### DISCUSSION TITLE
2007/09/25/%E5%A5%BD%E4%B8%80%E6%AE%B5%E6%97%B6%E9%97%B4%E6%B2%A1%E6%9B%B4%E6%96%B0%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#178] [Announcements]
# 2007/09/25/%E5%A5%BD%E4%B8%80%E6%AE%B5%E6%97%B6%E9%97%B4%E6%B2%A1%E6%9B%B4%E6%96%B0%E4%BA%86

最近一直没有更新我的 blog，大家提的问题也都没仔细看过。

https://blog.qizhen.xyz/2007/09/25/%E5%A5%BD%E4%B8%80%E6%AE%B5%E6%97%B6%E9%97%B4%E6%B2%A1%E6%9B%B4%E6%96%B0%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yoyo
2007年09月26日 23:22 编辑
^^

回复
竹贤
2007年09月26日 08:09 编辑
 抓紧啊，不要让我们失望。希望看到你的新文章。

回复
